### DISCUSSION TITLE
Is JsStore web site hacked?

### DISCUSSION THREAD

**Discussion by FlashGordon59** [ujjwalguptaofficial/JsStore#258] [Q&A]
Yesterday I got a bunch of warnings trying to connect to https://jsstore.net 

Does anybody know if the jsstore web site has been hacked?

**Comment by ujjwalguptaofficial** [comment]
No no, the website uses github-pages . So if someone will hack github then only jsstore site can be hacked.

Most probably github pages might be down. Sometime it happens.

**Reply by FlashGordon59** [reply]
Hi, thank you for you fast reply.. I think that my internet provider (WindTre) has decided that your site is potentially unsafe  and they are obscuring it ... the only way to see your site for me is downloading its contents from an exernal service and converting the pages in PDF files.

**Reply by FlashGordon59** [reply]
Last but no least ... from different providers I can see you site....
